src/components/learner-credit-management/BudgetDetailPageOverviewUtilization.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Stack, Collapsible, Row, Col, Button,
+} from '@edx/paragon';
+import { ArrowDownward } from '@edx/paragon/icons';
+import { useHistory } from 'react-router';
+
+import { generatePath, useRouteMatch } from 'react-router-dom';
+import { formatPrice } from './data';
+
+const BudgetDetailPageOverviewUtilization = (
+  {
+    budgetId,
+    budgetTotalSummary: { utilized },
+    budgetAggregates,
+    isAssignable,
+  },
+) => {
+  const history = useHistory();
+  const routeMatch = useRouteMatch();
+
+  const { amountAllocatedUsd, amountRedeemedUsd } = budgetAggregates;
+
+  if (budgetId === null || utilized <= 0 || !isAssignable) {
+    return null;
+  }
+
+  const handleTableSkipLink = (type) => {
+    const location = {
+      pathname: generatePath(routeMatch.path, { budgetId, activeTabKey: 'activity' }),
+      state: { budgetActivityScrollToKey: type },
+    };
+
+    history.push(location);
+  };
+
+  const renderActivityLink = ({ amount, type }) => {
+    if (amount <= 0) {
+      return null;
+    }
+
+    const linkText = (type === 'assigned') ? 'View assigned activity' : 'View spent activity';
+
+    return (
+      <Button
+        onClick={() => handleTableSkipLink(type)}
```

Comment:
It looks like it might be possible to rely on `Link` here as well, passing the location state via the `to` prop, ensuring the button acts more like a regular ol' hyperlink, e.g. 

```jsx
<Button
  as={Link}
  to={{
    pathname: generatePath(routeMatch.path, { budgetId, activeTabKey: 'activity' }),
    state: { budgetActivityScrollToKey: type },
  }}
  size="sm"
  variant="link"
  iconAfter={ArrowDownward}
  className="pl-0 pr-0"
>
  {linkText}
</Button>
```